docs/how-to/configure-virtual-ip-on-openstack.md

```diff
@@ -0,0 +1,58 @@
+# How to configure a virtual IP on openstack
```

Comment:
Out of curiosity: does this relate to this discourse post comment?
https://discourse.canonical.com/t/how-to-deploy-a-charm-on-an-existing-server/5122/5?u=amandahla

Reply:
In a way yes, in our case we create the openstack port but don't assign it to anything, the port is just there to "reserve" the IP address to prevent it from being assigned to another machine